### ISSUE TITLE
inconsistency in enum documentation

### ISSUE THREAD

**Issue by dlehmann** [php/doc-en#1780]
there is an inconsistency in the enum examples here

https://wiki.php.net/rfc/enumerations

This enum:

```php
enum SortOrder {
 case ASC;
 case DESC;
}
```
should also be PascalCase:

```php
enum SortOrder {
 case Asc;
 case Desc;
}
```

**Comment by cmb69** [comment]
Yeah, but I don't think it makes sense to "fix" this in the RFC. Or do you think otherwise, @Crell?

**Comment by salathe** [comment]
The same `enum` is used in the manual as well: https://www.php.net/manual/en/language.enumerations.examples.php

**Comment by Crell** [comment]
I'll fix the docs page.  I don't know if it's appropriate to update the RFC at this point, but if someone else did so I wouldn't raise a fuss.

**Comment by Girgias** [comment]
Fixed by #1785
